pkg/providers/cloudstack/cloudstack.go

```diff
@@ -87,9 +91,54 @@ func (p *cloudstackProvider) PostWorkloadInit(ctx context.Context, cluster *type
 }
 
 func (p *cloudstackProvider) UpdateSecrets(ctx context.Context, cluster *types.Cluster) error {
+	contents, err := p.createSecrets(ctx, cluster)
+	if err != nil {
+		return fmt.Errorf("creating secrets object: %v", err)
+	}
+
+	err = p.providerKubectlClient.ApplyKubeSpecFromBytes(ctx, cluster, contents)
+	if err != nil {
+		return fmt.Errorf("applying secrets object: %v", err)
+	}
 	return nil
 }
 
+func (p *cloudstackProvider) createSecrets(ctx context.Context, cluster *types.Cluster) ([]byte, error) {
+	if err := p.providerKubectlClient.CreateNamespaceIfNotExists(ctx, cluster.KubeconfigFile, constants.EksaSystemNamespace); err != nil {
```

Comment:
could we move this to the create workflow? 🙏🏻

Reply:
Moved to CreateBootstrapCluster in bootstrapper.go and removed creating eksa-system namespace from providers code.